src/app/core/models/netsuite/netsuite-configuration/netsuite-export-setting.model.ts

```
@@ -107,6 +107,9 @@ export class NetSuiteExportSettingModel extends ExportSettingModel {
     }
 
     static getCreditCardExportTypes(): SelectFormOption[] {
+      if (!brandingFeatureConfig.featureFlags.exportSettings.isReimbursableExpensesAllowed) {
+        return this.getCoCreditCardExportTypes();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)